lib/hanami/cli/command.rb

```diff
@@ -13,6 +14,7 @@ class Command
       def self.inherited(base)
         super
         base.extend ClassMethods
+        base.include Callbacks
```

Comment:
@davydovanton There is no need to allow callbacks DSL to a command.

Again, we **won't** allow this syntax:

```ruby
class Command < Hanami::CLI::Command
  before { ... }
  after { ... }
end
```

Including this `Callbacks` module here, will create public API that we **won't** support and need.